plugins/woocommerce/src/Internal/Admin/Settings/PaymentsRestController.php

```diff
@@ -188,6 +210,19 @@ protected function get_providers( WP_REST_Request $request ) {
 		return rest_ensure_response( $this->prepare_payment_providers_response( $response ) );
 	}
 
+	/**
+	 * Set the country for the payment providers.
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 *
+	 * @return WP_Error|WP_REST_Response
+	 */
+	protected function set_country( WP_REST_Request $request ) {
+		$location = $request->get_param( 'location' );
+		$this->payments->set_country( $location );
+		return rest_ensure_response( array( 'success' => true ) );
```

Comment:
If `set_country` returns a bool, we could:
```suggestion
		$location = $request->get_param( 'location' );
		
		$result = $this->payments->set_country( $location );
		
		return rest_ensure_response( array( 'success' => $result ) );
```